Makefile

```diff
@@ -240,6 +241,20 @@ build-plugin-admin-with-oci-discovery: ${CLI_ADMIN_JOBS_OCI_DISCOVERY} publish-a
 .PHONY: build-plugin-admin-with-local-discovery
 build-plugin-admin-with-local-discovery: ${CLI_ADMIN_JOBS_LOCAL_DISCOVERY} publish-admin-plugins-all-local ## Build Tanzu CLI admin plugins with Local standalone discovery
 
+.PHONY: build-plugin-local
+build-plugin-local: ## Build given CLI Plugin locally, needs PLUGIN_NAME
```

Comment:
This target needs to be linked with `prepare-builder`. I am seeing below error otherwise

```
$ make build-plugin-local PLUGIN_NAME=login
bash: /Users/anujc/Documents/code/tanzu-framework/bin/builder: No such file or directory
make: *** [build-plugin-local] Error 127
```

Reply:
for some reason, i don't see the error even in the new terminal, anyhow I have updated the target with prepare-builder

Reply:
The builder used by the Makefile is in `./bin/builder`.  If you have built it anytime by another make target, it will stay there.  This is probably why Chandra didn't see the error.